pkg/volume/cinder/attacher.go

```diff
@@ -75,9 +78,9 @@ func (attacher *cinderDiskAttacher) Attach(spec *volume.Spec, nodeName types.Nod
 
 	pending, volumeStatus, err := attacher.cinderProvider.OperationPending(volumeID)
```

Comment:
When OperationPending() failed to get volume, it return error and pending=false.
When OperationPending() get volume and volume's status is pending, it return pending=true/err=nil.
Or return pending=false/err=nil.